quantum/velocikey.c

```diff
@@ -24,6 +24,10 @@ void velocikey_toggle(void) {
         eeprom_update_byte(EECONFIG_VELOCIKEY, 1);
 }
 
+uint8_t get_typing_speed(void) {
```

Comment:
I don't think this is needed.  

If I'm reading things correctly, the `velocikey_match_speed` function should actually handle this.

In fact, I think you want to use `get_interval_time` instead. 

@chrislewisdev  Could you double check this?

Reply:
It's definitely not needed, I thought it was conceptually cleaner. My thinking was that `velocikey_match_speed` is a function that takes a unitless `typing_speed` and returns a value in milliseconds, and I want a function that takes a unitless `typing_speed` and returns a hue value. I'm noticing now that since I can clamp `velocikey_match_speed` to whatever values I like, it's a bit of a pointless distinction. I'll update to use `get_interval_time` instead, thanks for catching that